src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+import { AnyVenue, VenueTemplate } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./AdminVenues.scss";
+
+interface AdminVenuesProps {
+  venues: WithId<AnyVenue>[];
+}
+
+export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const partyVenues = useMemo(() => {
+    return venues?.filter((venue) => venue.template === VenueTemplate.partymap);
+  }, [venues]);
+
+  const hasVenues: boolean = !!partyVenues.length;
+
+  return (
+    <div className="admin-venue">
+      <div className="admin-venue__create-button">
+        <h3>Admin Dashboard</h3>
+        <Button as={Link} to="/admin_v2/venue/creation">
+          Create a new space
+        </Button>
+      </div>
+      {!hasVenues && (
+        <div className="admin-venue__empty-venues">
+          <h3>Welcome!</h3>
+          <h3>Create your first Sparkle space</h3>
```

Comment:
This feels like we're abusing the `h3` tag for its size/etc, but breaking it's 'semantic guarantee' of being a heading (which has impacts on accessibility/screenreaders/similar)

Reply:
Isn't this like the only reason to use it? For it's styling and heading purpose? Otherwise we'd be styling everything in the project and have no h1,h2....etc.

Reply:
Having 2 `h3` headings one after the other isn't using it for it's 'heading purpose'. We also already use styling explicitly to set text size/boldness/etc in many places within the app.

The heading tags are designed to convey the semantic structure of a document's sections, as well as it's hierarchical flow through subsections. The fact that they are styled a certain way is a side effect (that is often abused by web developers), rather than their main intent/purpose.

---

- https://www.w3.org/WAI/tutorials/page-structure/headings/
  - >  Headings with an equal or higher rank start a new section, headings with a lower rank start new subsections that are part of the higher ranked section.
- https://usability.yale.edu/web-accessibility/articles/headin
  - > Do not select heading levels based on their appearance. Select the appropriate heading rank in your hierarchy.
  - > Headings are often improperly used in cases where there are no dedicated semantic HTML elements(link is external). Headings should not be used as subheadings, subtitles, alternative titles, taglines, or slogans. In such cases, it is best to use a <p> tag.
 
---

It's the same reason that bootstrap separates its 'header-like styles' from the semantic h1-6 tags:

- https://getbootstrap.com/docs/4.0/content/typography/
  - > .h1 through .h6 classes are also available, for when you want to match the font styling of a heading but cannot use the associated HTML element.

Reply:
In some of those links there are examples with multiple headings one after another and I think it makes perfect sense. Why would it be semantically incorrect to have multiple h2 or h3 one after another. In my case I have 2 heading texts and I'm taking advantage of the h3. Yes, there are no children which makes it slightly confusing. And there are no parent elements h1 and h2. But I don't think using multiple headings one after another is a wrong thing to do.

Reply:
Those examples omit the content for brevity, but the intent is that they are each a unique section containing data. The way it’s used here is for the ‘same’ heading, or perhaps as a ‘subheading’.

@mcflyok May be outside the technical realm of ‘design’, but since you’re pretty up to scratch on your HTML demo skills, do you have any thoughts here?

@mike-lvov Potentially also?

Reply:
This still needs to be done.